src/zc/buildout/allow-unknown-extras.txt

```diff
@@ -0,0 +1,65 @@
+======================
```

Comment:
This is fine, however, the experiment to reuse doctests as documentation is over.   A regular unit test would have been just fine. :)